manifest.json

```diff
@@ -131,10 +131,17 @@
           "event": {
             "title": "Event Name",
             "type": "string",
-            "format": "event"
+            "format": "event",
+            "options": {
+              "standard": {
+                "ENTERED_USER_SEGMENT": "Entered User Segment",
+                "LEFT_USER_SEGMENT": "Left User Segment"
```

Comment:
Are we missing `ENTERED_ACCOUNT_SEGMENT` and `LEFT_ACCOUNT_SEGMENT` here ?

Reply:
@unity yes, I haven't added account segment support yet. I'll be working on that part later today.